apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -402,6 +425,36 @@ export class DotEditContentHtmlService {
         this.dotEditContentToolbarHtmlService.addContainerToolbar(doc);
     }
 
+    private injectInlineEditingScripts(): void {
+        const doc = this.getEditPageDocument();
+        const editModeNodes = doc.querySelectorAll('[data-mode]');
+
+        if (editModeNodes.length) {
+            const TINYMCE = `/html/js/tinymce/js/tinymce/tinymce.min.js`;
+            const tinyMceScript = this.dotDOMHtmlUtilService.creatExternalScriptElement(TINYMCE);
+            const tinyMceInitScript: HTMLScriptElement = this.dotDOMHtmlUtilService.createInlineScriptElement(
+                INLINE_TINYMCE_SCRIPTS
+            );
+
+            this.dotLicenseService
+                .isEnterprise()
+                .pipe(
+                    take(1),
+                    filter((isEnterprise: boolean) => isEnterprise === true)
+                )
+                .subscribe(() => {
+                    // We have elements in the DOM and we're on enterprise plan
+
+                    doc.body.append(tinyMceInitScript);
```

Comment:
so you're verifying enterprise license, but in the end you're always doing append? no matter what?

Reply:
@alfredo-dotcms  The code in the subscribe will never run if isEnterprise is false.

```javascript
of(false).pipe(
  filter(val =>  val === true)
).subscribe(console.log);
```